modules/LaneDetection/src/ImageProcessor.cpp

```diff
@@ -2,10 +2,12 @@
 
 namespace XyCar
 {
-    // void ImageProcessor::init()
-    // {
-    //     mask_lidar_ = cv::imread("../src/mask.png", cv::IMREAD_GRAYSCALE);
-    // }
+    void ImageProcessor::set_configuration(const YAML::Node& config)
+    {
+        k_roi_frame_y = config["IMAGE"]["ROI_Y_POS"].as<uint32_t>();
+        k_frame_width = config["IMAGE"]["WIDTH"].as<uint32_t>();
+        k_roi_frame_height = config["IMAGE"]["ROI_HEIGHT"].as<uint32_t>();
```

Comment:
```suggestion
        roi_frame_y_ = config["IMAGE"]["ROI_Y_POS"].as<uint32_t>();
        frame_width_ = config["IMAGE"]["WIDTH"].as<uint32_t>();
        roi_frame_height_ = config["IMAGE"]["ROI_HEIGHT"].as<uint32_t>();
```
더이상 상수는 아니니 k는 빼주는 것이 좋겠습니다 :)
전체 적용해주세요